### ISSUE TITLE
Disable correlation on a specific event

### ISSUE THREAD

**Issue by adulau** [MISP/MISP#1596]
Disable correlation on a specific event. If the feed feature is implemented, we might need to have a way to disable correlation on a specific event and (r)enable it if we want.

**Comment by MichaelDwucet** [comment]
I also could see a use case for disabling the correlation server wide for specific attributes.
E.g. Payload delivery - malware-type 
We have an enrichment module that will add, from an internal analysis system, amongst other things a malware type like "Trojan horse". So all our events that include the malware type "Trojan horse" get correlated. This is not very useful, cause the events are not connected, except for having a trojan horse as the malware type.

**Comment by iglocska** [comment]
Nice idea, that would indeed make a lot of sense. Also, additionally maybe malware-type shouldn't correlate in the first place?

**Comment by MichaelDwucet** [comment]
Yes, I can not think, in the normal MISP environment, why you would like to correlate all events with macro viruses, adwares, exploits, ...

To add to this: Is there are description of the correlation engine? Cause I did not know that distinct attributes cross correlate. An event with Payload delivery - malware-type - "Trojan Horse" gets correlated with an event with Antivirus detection - text - "Trojan Horse"

At the moment, as a workaround, we add $hash + value in some of the text fields, instead of just the value. Cause, we had multiple events with Antivirus detection - text - "Trojan.Generic" that got connected.
